<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Downloader</class>
 <widget class="QDialog" name="Downloader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Updating Application...</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>8</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="FileLabel">
       <property name="styleSheet">
        <string notr="true">color: rgb(147, 147, 147);</string>
       </property>
       <property name="text">
        <string>Waiting...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="fileProgressBar">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>10</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(119, 155, 255);
background-color: rgb(250, 250, 250);</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="ProcessLabel">
       <property name="styleSheet">
        <string notr="true">color: rgb(147, 147, 147);</string>
       </property>
       <property name="text">
        <string>Waiting...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="processProgressBar">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>10</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(119, 155, 255);
background-color: rgb(250, 250, 250);</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="infoLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(50, 50, 50);</string>
         </property>
         <property name="pixmap">
          <pixmap resource="../EResources.qrc">:/images/wait</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="restartButton">
         <property name="text">
          <string>Restart</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../EResources.qrc"/>
 </resources>
 <connections/>
</ui>
